tests/test.py

```diff
@@ -0,0 +1,192 @@
+import pytest
+import pandas as pd
+from pandas import read_csv
+from pathlib import Path
+
+from src.run_cocci import run_cocci
+from src.option_select import select
+
+### Modify this later
+patch_to_input_mapping = {
+    "assignment_as_value.cocci": ["assignment_as_value.c"],
```

Comment:
Instead of defining these mappings, I'd use `fixtures` to define paths to `cocci` files and input and expected files and would pass them into the test function. There would be a bunch of fixtures, but that's fine. It's cleaner